cabal.project

```diff
@@ -0,0 +1,4 @@
+packages: .
+debug-info: 3
```

Comment:
We don't need these options here. The person who builds an executable which uses the library needs them.